gtda/time_series/target.py

```diff
@@ -69,8 +69,7 @@ class Labeller(BaseEstimator, TransformerResamplerMixin):
      [7]
      [8]]
     >>> print(y)
-    [0.81649658 0.81649658 0.81649658 0.81649658 0.81649658 0.81649658
-     0.81649658 0.81649658]
+    [0 1 2 3 4 5 6 7]
```

Comment:
This doesn't seem right: shouldn't `y[0]` be equal to `max([0,1,2])`.
I think there might have been some issues with mutability of `X`? I get a different result (the one I would expect), if I rename the variable:

```
>>> import numpy as np
>>> from gtda.time_series import Labeller
X>>> X = np.arange(10)
>>> labeller = Labeller(width=2, func=np.max)
>>> x_r, y_r = labeller.fit_transform_resample(X, X)
>>> x_r
array([[1],
       [2],
       [3],
       [4],
       [5],
       [6],
       [7],
       [8]])
>>> y_r
array([2, 3, 4, 5, 6, 7, 8, 9])
```

Reply:
I wanted to write `func=np.min` but wrote `np.max` by mistake! The output is the one shown in that case.  I'm not sure what the issues with mutability are that you are experiencing. It's true that both `transform` and `resample` here return views, and at least for `transform` we should probably return copies instead!

Reply:
Ok, the explanation is straightforward

Reply:
@wreise the typo is now fixed.